datafeeds/datafeed_fms_api.py

```diff
@@ -310,9 +315,18 @@ def getEventRankings(self, event_key):
     def getTeamDetails(self, year, team_key):
         team_number = team_key[3:]  # everything after 'frc'
 
-        result = self._parse(self.FMS_API_TEAM_DETAILS_URL_PATTERN % (year, team_number), FMSAPITeamDetailsParser(year))
+        avatar = self._parse(self.FMS_API_TEAM_DETAILS_URL_PATTERN % (year, team_number), FMSAPITeamDetailsParser(year))
```

Comment:
`avatar` > `avatars` (and the 2 lines below)

Reply:
Even though it's only ever going to be one?  🤔

Reply:
It's a list of 1, which is why the next line is `avatars[0]`

Reply:
Fair point.  Also, the fact that you called that out made me realize I changed the `getTeamDetails` method, not the `getTeamAvatar` method right below it... 😅